lib/SPIRV/SPIRVReader.cpp

```diff
@@ -987,6 +995,41 @@ void SPIRVToLLVM::setLLVMLoopMetadata(const LoopInstType *LM,
   }
   if (LC & LoopControlNoFusionINTELMask)
     Metadata.push_back(getMetadataFromName("llvm.loop.fusion.disable"));
+
+  if (LC & spv::internal::LoopControlLoopCountINTELMask) {
```

Comment:
I believe here we need a comment about size of LoopCountINTELMask parameters (int64) and that they are stored as SPIRVWords (int32).

Reply:
Added comment in [a2d546c](https://github.com/KhronosGroup/SPIRV-LLVM-Translator/pull/1062/commits/a2d546cfaa70d488cdcd44be1a1ef57c08f076ca)